Fix in description of Learn Accessibility by Building a Quiz - Step 41 (#49530)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145f685797bd30df9784e8c.md

# --description--

> Do not forget to give your `form` a submit button. The submit should display the text "Send".

Reviewer: ```suggestion
Do not forget to give your `form` a submit button with `Send` as its text.
```
